docs/reference/announcements/880.md

```diff
@@ -143,7 +143,8 @@ The `CamundaClient` replaces the `ZeebeClient`, offering the same functionality
 
 :::note
 
-- If you need to continue using the old `ZeebeClient`, you can use the version 8.6 artifact without any issues with newer cluster versions as the client is forward-compatible.
+- If you need to continue using the old `ZeebeClient`, you can use the new version 8.8 `CamundaClient` artifact without issues as it still contains the related `ZeebeClient` classes. Those classes are marked as deprecated, so you can easily spot code you need to adjust to the `CamundaClient`.
```

Comment:
We have this noted for 860.md and here in 880.md for the next version of docs, but no mention of this in 870.md -- this should be included there as well, yes?

Lastly, this is a deprecation but not necessarily a breaking change, yes?